testing/run-scripts/run_cloud.sh

```diff
@@ -87,89 +93,103 @@ fi
 #  - pull from DNS
 if [ -z "$PUBLIC_IP" ]
 then
-  if docker ps -a | grep uproxy-sshd >/dev/null
-  then
-    if [ `docker inspect --format='{{ .State.Status }}' uproxy-sshd` != "running" ]
+    if docker ps -a | grep uproxy-sshd >/dev/null
+    then
+        if [ `docker inspect --format='{{ .State.Status }}' uproxy-sshd` != "running" ]
+        then
+            docker start uproxy-sshd > /dev/null
+        fi
+        # Don't fail if the current installation has no /hostname.
+        PUBLIC_IP=`docker exec uproxy-sshd cat /hostname || echo -n ""`
+    fi
+    if [ -z "$PUBLIC_IP" ]
     then
-      docker start uproxy-sshd > /dev/null
+        # Beautiful cross-platform one-liner cogged from:
+        #   http://unix.stackexchange.com/questions/22615/how-can-i-get-my-external-ip-address-in-bash
+        PUBLIC_IP=`dig +short myip.opendns.com @resolver1.opendns.com`
     fi
-    # Don't fail if the current installation has no /hostname.
-    PUBLIC_IP=`docker exec uproxy-sshd cat /hostname || echo -n ""`
-  fi
-  if [ -z "$PUBLIC_IP" ]
-  then
-    # Beautiful cross-platform one-liner cogged from:
-    #   http://unix.stackexchange.com/questions/22615/how-can-i-get-my-external-ip-address-in-bash
-    PUBLIC_IP=`dig +short myip.opendns.com @resolver1.opendns.com`
-  fi
 fi
 
-if [ $REFRESH = true ]
+if [ -z "$INVITE_CODE" ]
+then
+    if docker ps -a | grep uproxy-sshd >/dev/null
+    then
+        if [ `docker inspect --format='{{ .State.Status }}' uproxy-sshd` != "running" ]
+        then
+            docker start uproxy-sshd > /dev/null
+        fi
+    fi
+    if [ $WIPE = false ]
+    then
+        INVITE_CODE=`docker cp uproxy-sshd:/initial-giver-invite-code -|tar xO`
```

Comment:
Let's add a `|| echo -n ""`` for the case where that file doesn't exist, like we did at line 103.